### PR TITLE
Update the packages needed for Isocortex

### PR DESCRIPTION
**Pull request by GianlucaFicarelli**
Update the packages needed for Isocortex

New versions:
- placement-algorithm==2.1.1 (update)
- py-diameter-synthesis==0.1.7 (new)
- py-region-grower==0.1.10 (update)
- py-tmd==2.0.9 (update)
- py-tns==2.2.7(update)

### CODE REVIEW

**Review by arnaudon — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-diameter-synthesis/package.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from spack import *
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/py-diameter-synthesis/package.py:6]: standard copyright header missing?

**Review by pramodk — CHANGES_REQUESTED** [review]
If you could fix copyright header, we are good to go!

File: var/spack/repos/builtin/packages/py-diameter-synthesis/package.py (hunk 1 above)
Comment (GianlucaFicarelli) [reply var/spack/repos/builtin/packages/py-diameter-synthesis/package.py:6]: My bad, I was thinking that the standard copyright didn't apply to new files used for internal packages. I'll add it back.

File: var/spack/repos/builtin/packages/py-tmd/package.py [hunk 2]
```diff
@@ -10,16 +10,17 @@ class PyTmd(PythonPackage):
     """Python library for the topological analysis of neurons"""
 
     homepage = "https://github.com/BlueBrain/TMD"
-    url = "https://pypi.io/packages/source/t/tmd/tmd-2.0.8.tar.gz"
+    url = "https://pypi.io/packages/source/t/tmd/tmd-2.0.9.tar.gz"
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-tmd/package.py:13]: You don't need to update the URL with every new version, /me thinks.

**Review by matz-e — APPROVED** [review]
Looks legit, thanks!

**Review by pramodk — APPROVED** [review]
lgtm!

**Pull request merged**